README.md

```diff
@@ -1,2 +1,21 @@
 # django-assets
 Assets management system
+
+## Getting Started
+
+### Dependencies
+
+ - Docker*
+ - Docker-Compose
+ - Make
+
+*Configuring non-root access is recommanded
+
+### Starting Development Environement
+
+ - run `$ make` in root project directory.
+ - Access the app via your browser at 0.0.0.0:8000
```

Comment:
The app IP is not 0.0.0.0, it's localhost. 0.0.0.0 when you launch the app means that it listen on all interfaceas